pkg/proto/v0/accounts.proto

```diff
@@ -21,8 +21,31 @@ message Payload {
 }
 
 message Account {
-    string name = 1;
-    string dateOfBirth = 2;
+    StandardClaims standard_claims = 1;
+}
+
+// OIDC standard claims https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims
+message StandardClaims {
```

Comment:
What is the use case for these claims?

Reply:
since this service is poorly specified, so in order to get something rolling we're adding the standard claims to the accounts. The idea is after log in we store the standard claims on the accounts service. It's just so we have some data to roll.

I think this service is a moving target and requires more specs.

Reply:
a different more permissive approach would be using `any` and let services store any information they want on the user, but I'm not fond of that approach, since versioning is lost, and so is type safety.

Reply:
So it is dummy / examples data as of now .... fine by me for now as long as it goes away at some point in time ..... 

User information is provided by the user backend (ldap or something) - nobody cares about these claims in out context.

At least I don't ;-)